app/models/chess_piece.rb

```diff
@@ -1,6 +1,6 @@
 # Chess piece model
 class ChessPiece < ActiveRecord::Base
-  BOARD_START_INDEX = 1
+  BOARD_BGN_INDEX = 1
```

Comment:
Begin...just was hoping to get the constants to line up nicely...not a big deal. :+1: